### PR TITLE
Add dough and filling option to pie gen

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

**Comment by sgomes** [comment]
The changes LGTM, but I think `src/shared/js/routes/piegen.js` and `src/shared/js/models/pie.js` should be removed, right? Carry-over from last PR.

Also, how about the `pie.hbs` and `pie.svg.hbs` views?

File: data/ingredients.json [hunk 1]
```diff
@@ -6,11 +6,18 @@
   "filling": {
     "cream": "#F5F1CB",
     "curd": "#F0D6A3",
-    "mousse": "#89795C"
+    "mousse": "#89795C",
+    "fudge": "#DfA84F",
```
Comment (sgomes) [inline data/ingredients.json]: Please be consistent with color codes; either caps or lowercase. Ideally, throughout the entire file.

**Comment by devnook** [comment]
I'm using src/shared/js/routes/piegen.js route for testing the pie generator, demoing it and work on pie loading temporarily. Once the real routes land in repo, it can be removed (but please not before). pie.hbs and pie.svg.hbs are used in the router atm so they should stay as they are.

**Comment by sgomes** [comment]
Cool, sounds good to me, then!

**Pull request merged**